notebooks/00_quick_start/rlrmc_movielens.ipynb

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
<p>In the other notebooks we have the parameters as one of the first cells, so I would recommend to move this up. We also test all our notebooks, the instructions are described here: <a href="https://github.com/Microsoft/Recommenders/tree/master/tests#how-to-create-tests-on-notebooks-with-papermill" target="_blank">https://github.com/Microsoft/Recommenders/tree/master/tests#how-to-create-tests-on-notebooks-with-papermill</a>, but the idea is simple, in the parameters cell we add a tag called `parameters`. Then from papermill, which is the program that executes the notebooks we inject the variables we want. For example, for the unit tests we will probably want to inject the size of movielens 100k, so the test is quick. </p><p>Please let me or any other of the core developers know if you need any help</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/729/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I have added the parameters tag via papermill. Resolving the issue. Please let me know if anything more is needed here.